config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  resources :customer_items
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get "/customers/:id", to: "customers#show"
+
+  get "/items", to: "items#index"
+  
```

Comment:
lines 4 and 6 are good restful routes. Line 2 will work, but you'll want to limit any undeveloped routes using an `only: [:create]` statement.